src/modules/Security.rb

```diff
@@ -597,12 +520,136 @@ def WriteConsoleShutdown(ca)
       true
     end
 
+    # Write the settings from @Locations to the corresponding files
+    def write_to_locations
+      commitlist = []
+      # NOTE: the call to #sort is only needed to satisfy the old testsuite
+      @Locations.sort.each do |file, vars|
+        vars.each do |var|
+          val = @Settings[var]
+          if val && val != SCR.Read(path("#{file}.#{var}"))
+            SCR.Write(path("#{file}.#{var}"), val)
```

Comment:
that not nice way to construct path, better is `path(file) + var` see https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/path.rb#L24